src/qt/optionsmodel.h

```diff
@@ -41,7 +41,7 @@ class OptionsModel : public QAbstractListModel
     Q_OBJECT
 
 public:
-    explicit OptionsModel(QObject *parent = nullptr, bool resetSettings = false);
+    explicit OptionsModel(interfaces::Node& node, QObject *parent = nullptr, bool resetSettings = false);
```

Comment:
It seems to not be strictly necessary to have to initialize `OptionsModel` with a reference to `interfaces::Node` and instead could just be set at a later point which is similar to how we do it now. Wondering what is the justification to add this to the constructor?

Reply:
nevermind, I see how this is needed in consideration of future changes :D